### PR TITLE
11573 A bunch of unit tests

### PR DESCRIPTION
**Pull request by severinbeauvais**
*Issue #:* /bcgov/entity#11573

*Description of changes:*
- deleted unneeded code
- changed some ids to classnames
- updated unit tests
- app version = 3.5.10

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the bcrs-entities-create-ui license (Apache 2.0).

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "business-create-ui",
-  "version": "3.5.9",
+  "version": "3.5.10",
```
Comment (severinbeauvais) [inline package.json:3]: This PR isn't only unit tests so I updated this app version -- so we can tell the difference if there's a bug in this version but not in the previous one.

File: src/views/Registration/RegistrationReviewConfirm.vue [hunk 2]
```diff
@@ -149,23 +149,11 @@ export default class RegistrationReviewConfirm extends Vue {
...
-      const emails = orgPersonList
-        .filter(op => op.roles.some(role => role.roleType === RoleTypes.PROPRIETOR))
-        .map(op => op.officer?.email)
-        .filter(e => !!e)
-      return emails.join(',')
-    }
-
-    if (this.isTypePartnership) {
-      const emails = orgPersonList
-        .filter(op => op.roles.some(role => role.roleType === RoleTypes.PARTNER))
```
Comment (severinbeauvais) [inline src/views/Registration/RegistrationReviewConfirm.vue:162]: I _think_ a "partner" is just a proprietor for a GP.

Or maybe we will need a new role type after all. I don't know. It's a future ticket.

**Review by cameron-freshworks — APPROVED** [review]
Looks good 👍

File: src/views/Registration/RegistrationReviewConfirm.vue (hunk 2 above)
Comment (severinbeauvais) [reply src/views/Registration/RegistrationReviewConfirm.vue:162]: @vysakh-menon-aot Any thoughts ^^ ?

**Pull request merged**